java/src/jmri/jmrit/beantable/BlockTableAction.java

```diff
@@ -139,7 +139,12 @@ public String getValue(String name) {
                 }
                 Object m = b.getValue();
                 if (m != null) {
-                    return m.toString();
+                    if ( m instanceof jmri.Reportable) {
+                        return ((jmri.Reportable) m).toReportString();
+                    }
+                    else {
+                        return m.toString();
```

Comment:
Since we know the object is a Bean, shouldn't we use getDisplayName instead of toString?

Reply:
OH wait, I forgot that Blocks can contain other objects, like strings... so getDisplayName could be applied for beans, but it isn't always applicable